wiki/Beatmap_Discussion/en.md

```diff
@@ -1,225 +1,183 @@
 
-### Returning to the old modding system
+It's important to keep a few things in mind to use the Beatmap Discussion pages effectively:
 
-It is no longer possible to return to the old modding system. The only beatmaps that may use the old system are those with mod posts. If beatmaps already use Beatmap Discussion, the forum thread will not have a reply button, instead, it will replaced with a button that redirects to the Beatmap Discussion panel.
-
-## Accessing the Beatmap Discussion page
-
-### Through new.ppy.sh:
+- Choose the correct mode and difficulty you wish to mod.
+- Select the proper section you are submitting your mod to.
+- **Only post one issue at a time.** Do not submit multiple issues in a single post.
+- Check to see if the issue has already been addressed when the similar post warning pops up.
+- Do you like the beatmap? Give your hype to help the beatmap move closer to nomination.
 
-Click on the `Discussion` button in the beatmap info page.
+## Mode selection
 
-![](img/info.jpg "Beatmap info page")
+![](img/mode-selection.jpg "Mode selection.")
 
-### Through osu.ppy.sh:
+Mode selection changes the game mode between the different modes: [osu!standard](/wiki/Game_Modes/osu!), [osu!taiko](/wiki/Game_Modes/osu!taiko), [osu!catch](/wiki/Game_Modes/osu!catch), and [osu!mania](/wiki/Game_Modes/osu!mania). It is only possible to select a game mode if the beatmapset has a difficulty of that game mode. This feature is mainly used for beatmaps which contain difficulties of multiple game modes. For normal beatmaps this remains untouched, since the discussion page automatically switches to the current game mode.
 
-Clicking on `Modding/Discussion` on the beatmap's page will take you to the page, if it's currently in the Beatmap Discussion system.
+## Beatmap header
 
-## Beatmap Discussion page
+![](img/beatmap-header.jpg "Beatmap header.")
 
-![](img/beatmap-discussion.jpg "The top portion of the Beatmap Discussion page")
+The beatmap header displays the Title and Artist-name provided in the [beatmap editor](/wiki/Beatmap_Editor). Clicking the beatmap header will navigate back to the beatmap info page.
 
-### Mode selection
+To see correct formatting, romanization and potential tags to be added to the title and artists fields, see the [Ranking Criteria](/wiki/Ranking_Criteria#metadata).
 
-![](img/modes.jpg "Mode selection")
+## Difficulty menu
 
-Changes the game mode between osu!standard, osu!taiko, osu!catch, and osu!mania. You can only select each game mode if the beatmapset has a difficulty with said game mode.
+![](img/difficulty-menu.jpg "Difficulty menu.")
 
-## Beatmap page / Difficulty selection
+You can select different [difficulties](/wiki/Difficulties) through the dropdown menu. This will show all currently available difficulties, along with older deleted ones that have had suggestions/problems. The number that appears besides a difficulty's name is the number of unresolved posts on that specific difficulty.
```

Comment:
```suggestion
You can select different [difficulties](/wiki/Difficulties) through the dropdown menu. This will show all currently available difficulties, along with older-deleted ones that have had suggestions/problems. The number that appears besides a difficulty's name is the number of unresolved posts on that specific difficulty.
```